staging/src/k8s.io/apiextensions-apiserver/pkg/cmd/server/testing/testfiles/localhost_127.0.0.1_localhost.crt

```diff
@@ -0,0 +1,37 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
do we have to do anything special to get bazel to pick these up (like name the folder testdata or something)?

Reply:
Very good point. Should be the same as in serving_test.go (if integration tests and unit tests work the same in bazel).

Reply:
@mikedanese @ixdy what's the best way to do this? We need the testdata in all tests which (indirectly) import apiextensions-apiserver/pkg/cmd/server/testing (similar for kube-apiserver).

Reply:
If you name a directory `testdata`, I believe `gazelle` will include it as a `data` dependency on all `go_test`s in that package, though that isn't quite enough for what you want here, it sounds like?

The canonical bazel way would be to create a filegroup for those data files, and then explicitly list them as `data` dependencies in each test that needs them. `gazelle` can't figure out what packages those would be, however. 

You can try adding a `data` dependency on the relevant `go_library` itself (i.e. https://github.com/sttts/kubernetes/blob/3ad9a14ae479536ba8b62dc77e6981117f0fdeaa/cmd/kube-apiserver/app/testing/BUILD#L8), which might do the right thing, adding a transitive `data` dependency to any tests that depend on this library. I haven't tried it, but it's worth a shot.

Reply:
Integration tests are not bazel based, are they? So for the moment we should be fine it seems.

Reply:
Nevertheless, I added the testdata dependency to the `testing` go library target.